modules/jfx.incubator.richtext/src/main/java/jfx/incubator/scene/control/richtext/model/SimpleViewOnlyStyledModel.java

```diff
@@ -93,10 +95,12 @@ public RichParagraph getParagraph(int index) {
 
     /**
      * Appends a text segment to the last paragraph.
-     * The {@code text} cannot contain newline ({@code \n}) symbols.
+     * The caller must ensure that the {@code text} does not contain newline symbols, as the behavior might be
+     * undefined.
```

Comment:
Minor: I would say "is undefined" (instead of "might be") meaning that we don't define or specify the behavior.